test/test_sailthru.js

```diff
@@ -586,5 +586,46 @@
     });
   };
 
+  // ensure we will correctly handle 201 and 204 responses
+  exports.saveWatch = function(test) {
+    const watchData = { 
+      profile_id: '12345', 
+      watch: { 
+        interest_type: 'wishlist', 
+        query: { content_id: '67890' },
+      },
+    }
+
+    nock('https://api.sailthru.com')
+      .post(/^\/content\/watch/, function (q) {
+        var data = JSON.parse(q.json);
+        test.deepEqual(data, watchData);
+        return true;
+      })
+      .reply(201, {/* don't care about response */});
+
+    test.expect(2);
```

Comment:
It's not clear to me as to why we are expecting 2 here? Shouldn't it be 1?

Reply:
Because we are only creating 1 item?

Reply:
The number is how many assertions nodeunit should expect. I've mainly just done this to follow the pattern elsewhere in this repo :)

Reply:
Ah right okay. I misunderstood that! thanks